joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

front_left_steer_controller:
  type: effort_controllers/JointPositionController
  joint: front_left_steer_joint
  pid: {p: 8.0, i: 0.0, d: 0.4}

front_right_steer_controller:
  type: effort_controllers/JointPositionController
  joint: front_right_steer_joint
  pid: {p: 8.0, i: 0.0, d: 0.4}

rear_left_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: rear_left_wheel_joint
  pid: {p: 0.1, i: 0.0, d: 0.0}

rear_right_wheel_controller:
  type: effort_controllers/JointVelocityController
  joint: rear_right_wheel_joint
  pid: {p: 0.1, i: 0.0, d: 0.0}
